### PR TITLE
common ui componets are added

### PR DESCRIPTION
**Pull request by Anilkumar-GA**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+
```
Comment (mkiyappan) [inline App.js:2]: Please remove this space

File: App.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import TextField from './src/components/TextField/TextField';
+import Button from './src/components/Button/Button';
+import Alert from './src/components/Alert/Alert';
+import Chip from './src/components/Chip/Chip';
+import Progress from './src/components/Progress/Progress';
+import Typography from './src/components/Typography/Typography';
+import Textarea from './src/components/Textarea';
+const App = () => {
```
Comment (mkiyappan) [inline App.js:18]: Add one space before starting component

File: App.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    const [value, setValue] = useState('');
+    const handleChange = (e) => {
+        console.log('handleChange value=>', e.target.value);
+    };
+    return (
+        <div>
+            {/* <TextField onChange={handleChange} varient="large" id="cdcf" /> */}
```
Comment (mkiyappan) [inline App.js]: Please remove  commented lines before commit

File: package.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "intelops_ui",
+  "version": "1.0.0",
+  "description": "create Hello\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C WOrld React App",
```
Comment (mkiyappan) [inline package.json]: Please update description

File: package.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open --hot",
+    "build": "webpack --mode production"
+  },
+  "author": "Jaspin Suji",
```
Comment (mkiyappan) [inline package.json:10]: Please update author name to "Intelops Team"

File: src/components/Alert/Alert.jsx [hunk 6]
```diff
@@ -0,0 +1,145 @@
+import React, { useEffect, useState } from 'react';
+import '../../assets/style/soft-ui-dashboard-tailwind.css';
```
Comment (mkiyappan) [inline src/components/Alert/Alert.jsx]: Local reference should be imported after node package

File: src/components/Alert/Alert.jsx [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+import PropTypes from 'prop-types';
+
+const Alert = (props) => {
+    const [alertVarient, setAlertVarient] = useState('relative w-full p-4 mb-4 text-white border');
+    const [closeAlert, setCloseAlert] = useState(false);
+
```
Comment (mkiyappan) [inline src/components/Alert/Alert.jsx:9]: Please remove unused space

File: src/components/Alert/Alert.jsx [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+            default:
+                setAlertVarient(`${alertVarient} border-solid rounded-lg bg-gradient-to-tl from-purple-700 to-pink-500 border-fuchsia-300`);
+        }
+    }, [varient]);
+
+    return (
+        <>
+            {!closeAlert ? (
+                <div
+                    alert
```
Comment (mkiyappan) [inline src/components/Alert/Alert.jsx:51]: Please add alert = "true"

File: src/components/Alert/Alert.jsx [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+                    id={id}
+                    className="grid place-items-center w-full min-h-[140px] bg-[#f8fafc] p-6 border border-blue-grey-50 rounded-lg scroll-mt-48 overflow-x-scroll lg:overflow-visible"
+                >
+                    <div alert class={alertVarient}>
```
Comment (mkiyappan) [inline src/components/Alert/Alert.jsx:55]: Please add alert="true"

File: src/components/Alert/Alert.jsx [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+                        {children}
+                        <button
+                            alert-close
```
Comment (mkiyappan) [inline src/components/Alert/Alert.jsx:58]: Pelase add alert-close="true"

File: src/components/Alert/index.js [hunk 11]
```diff
@@ -0,0 +1 @@
+export { default } from './Typography';
```
Comment (mkiyappan) [inline src/components/Alert/index.js:1]: Please remove this index file in alert folder

File: src/components/Button/Button.jsx [hunk 12]
```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```
Comment (mkiyappan) [inline src/components/Button/Button.jsx]: Please remove this font awesome package as discussed

File: src/components/Button/Button.jsx [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+import { faList } from '@fortawesome/free-solid-svg-icons';
```
Comment (mkiyappan) [inline src/components/Button/Button.jsx]: Please remove this font awesome package as discussed

File: src/components/checkbox/checkbox.jsx [hunk 14]
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react'
+import "../../style/soft-ui-dashboard-tailwind.css";
+
+const Checkbox = () => {
+    const [checked, setChecked] = useState(false);
+    const handleChange = () => {
```
Comment (mkiyappan) [inline src/components/checkbox/checkbox.jsx:6]: Call back is missing..pls check

File: src/components/dropdown/dropdown.jsx [hunk 15]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import "../../style/soft-ui-dashboard-tailwind.css";
+import "../../js/soft-ui-dashboard-pro-tailwind";
+
+const Dropdownlist = () => {
+
+    //const [open, setOpen] = React.useState(false);
```
Comment (mkiyappan) [inline src/components/dropdown/dropdown.jsx:7]: Please set props here and set default props as well

File: src/components/modal/modal.jsx [hunk 16]
```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import "../../style/soft-ui-dashboard-tailwind.css";
+import "../../js/soft-ui-dashboard-pro-tailwind";
+
+const Modal = () => {
+
+    return (
```
Comment (mkiyappan) [inline src/components/modal/modal.jsx:7]: Please make this component dynamic..

**Comment by iyappanmk** [comment]
merge

**Review by iyappanmk — APPROVED** [review]

**Pull request merged**